### DISCUSSION TITLE
`src-tauri/tauri.conf.json`文件中没有`transparent `

### DISCUSSION THREAD

**Discussion by himonster1** [tw93/Pake#456] [Q&A]
https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E5%BC%80%E5%8F%91%E8%80%85%E4%BD%BF%E7%94%A8%EF%BC%89#5-%E9%AB%98%E7%BA%A7%E9%85%8D%E7%BD%AE

高级设置中提到的这个文件里面没有transparent

**Comment by tw93** [comment]
抱歉 这个文档我最近更新一下，你说的这个在pake.json里面了
